procurement_purchase_no_grouping/models/res_company.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2020 - Radovan Skolnik <radovan@skolnik.info>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    procured_purchase_grouping = fields.Selection(
+        [
+            ("standard", "Standard grouping"),
+            ("line", "No line grouping"),
+            ("order", "No order grouping"),
+        ],
+        string="Procured purchase grouping",
+        default="standard",
+        required=True,
```

Comment:
It's better to avoid this `required`, as in tests, this will be a nuisance, as this constraint is put on DB, but default is not applying until next run, so it provokes failures. Instead, put `required="1"` in the res_config_settings view for UI.

Reply:
But that would mean that unless someone sets it, it will be None? In combination with that Product Category does not have a default value we can run into situation where there will be no value at all - i.e. directly after installation.

Reply:
No, required != default. The default will still be there. This is only a technical question for not setting the NOT NULL constraint on the DB, so you don't have problems in tests.

Reply:
Still learning new thing. Thanx a lot. Will do.